docs/api/web-contents.md

```diff
@@ -1168,6 +1168,7 @@ Ignore application menu shortcuts while this web contents is focused.
     * `url` string - The _resolved_ version of the URL passed to `window.open()`. e.g. opening a window with `window.open('foo')` will yield something like `https://the-origin/the/current/path/foo`.
     * `frameName` string - Name of the window provided in `window.open()`
     * `features` string - Comma separated list of window features provided to `window.open()`.
+    * `parsedFeatures` Record<string, any> - Parsed version of the features.
```

Comment:
Bikeshedding: The [MDN window.open article](https://developer.mozilla.org/en-US/docs/Web/API/Window/open) uses the name `windowFeatures`. What do you think about renaming this? It reminds me a bit of the `url.search` and `url.searchParams` APIs.